README.md

```diff
@@ -181,6 +181,45 @@ ENV BROKER_CLIENT_URL   http://my.broker.client:8000
 ENV PORT                8000
 ```
 
+### Azure Repos
+
+To use the Broker client with [Azure](https://azure.microsoft.com/en-us/services/devops/), run `docker pull snyk/broker:azure-repos` tag. The following environment variables are mandatory to configure the Broker client:
+
+- `BROKER_TOKEN` - the Snyk Broker token, obtained from your Azure Repos integration settings view (app.snyk.io).
+- `AZURE_REPOS_TOKEN` - an Azure Repos personal access token
```

Comment:
Do we need to define what permissions are required here? We seem to for other SCMs

Reply:
Mentioned this in Dockerfile. Worth to add here as well